libs/cardano-ledger-core/src/Cardano/Ledger/Tools.hs

```diff
@@ -0,0 +1,124 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeOperators #-}
+
+-- | This is a module that contains functionality that is not necessary for ledger
+-- operation, but is usefuk for testing as well as for downstream users of ledger
```

Comment:
I assume this should say `useful` ? :laughing:

Reply:
I think so too :laughing: